TrueCraft.API/Ray.cs

```diff
@@ -5,12 +5,24 @@
 
 namespace TrueCraft.API
 {
-    // Mostly taken from the MonoXna project, which is licensed under the MIT license
```

Comment:
Done, is it possible to put the commit with this pull request or do I have to open another?